www/app.py

```diff
@@ -36,5 +44,37 @@ def getblockdiagram(id):
         content = content_file.read()
     return Response(content, mimetype='text/xml')
 
+@app.route('/api/v1/sendcommand', methods = ['POST'])
+def sendCommand():
+    print request.form
+    runCommand(request.json)
+    return jsonify(request.json)
+
+def init_rover_service():
```

Comment:
I agree: lowercase with underscores. 
Let's make this a separate "cleanup" commit, since it isn't related to this feature.